src/types/api/schema.ts

```diff
@@ -85,6 +85,81 @@ export interface TDirEntry {
     Version?: TPathVersion;
 }
 
+interface TDomainDescription {
```

Comment:
I think some fields from the protobuf are missing. Is it intentional?

Reply:
Added this missing types

Reply:
ok, I think this is fine, but we may want to mark these interfaces as incomplete, like some others currently are marked